AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -450,10 +457,10 @@ async function runConnectivityCheckAsync(hostname, port, dnsServers, diagProvide
 }
 
 async function validateConnectionStringAsync(propertyName, connectionString, type, diagProvider, failureDetailsMarkdown = undefined) {

```

Comment:
This method is no longer called from anywhere in the code and can be removed.  Please confirm and remove and please try to check for things like this to avoid leaving dead code around.  For C# VS does a great job or highlighting issues like this but for js we need to check ourselves or find a tool that is equivalent to VS for js.